studio/components/interfaces/Settings/Logs/PreviewFilterPanel.tsx

```diff
@@ -84,7 +84,7 @@ const PreviewFilterPanel: FC<Props> = ({
             >
               <div className="absolute z-20">
                 <Typography.Text style={{ fontSize: '0.6rem' }} className="opacity-80">
-                  {newCount}
+                  {newCount > 1000 ? `${Math.floor(newCount / 100) / 10}K` : newCount}
```

Comment:
Could also have "100+" maybe - up to you. approving anyway.

Reply:
hmm i think seeing the absolute number rising is more gratifying for the user 😄